lib/service_sign_in_yaml_validator.rb

```diff
@@ -19,6 +20,7 @@ def valid?
     load_yaml_file
     return unless @yaml_file.present?
     check_for_top_level_required_fields
+    check_for_choose_sign_in_required_fields if choose_sign_in.present?
```

Comment:
Shouldn't this error if choose_sign_in isn't present first as choose_in_sign itself is required?

Reply:
If `choose_sign_in` isn't present it will be picked up in `check_for_top_level_required_fields` and already logged as an error.

Reply:
Sorry. I missed that!

Reply:
If `choose_sign_in` isn't present that we don't want to call `check_for_choose_sign_in_required_fields` as it'll blow up, so we return in this scenario.